Zend/zend_API.c

```diff
@@ -3500,7 +3500,7 @@ ZEND_API void zend_release_fcall_info_cache(zend_fcall_info_cache *fcc) {
 	}
 }
 
-static zend_always_inline bool zend_is_callable_check_func(zval *callable, zend_execute_data *frame, zend_fcall_info_cache *fcc, bool strict_class, char **error) /* {{{ */
+static zend_always_inline bool zend_is_callable_check_func(zval *callable, zend_execute_data *frame, zend_fcall_info_cache *fcc, bool strict_class, char **error, bool suppress_deprecation) /* {{{ */
```

Comment:
Maybe put the `suppress_deprecation` parameter before `error`, since the latter is an out-parameter.

Reply:
But that would be inconsistent with existing `zend_is_callable_check_class` (ref: https://github.com/php/php-src/commit/ee510eed685cee62cd99a210e27bdee6892861e3#diff-210c47b94a106559d6926f51c60d8c0d39f996738af3fe79e983a409d87a6b45L3294-R3294)?

Reply:
Indeed, that's why I initially put it where I did; however, I think both functions are entirely internal, so I'm tempted to rearrange both, as out params on the end does feel sensible.

Reply:
Then note: the parameter currently before `error`, `strict_class`, is "in" here but "out" there

Reply:
Well, maybe then leave this as is now.